### DISCUSSION TITLE
Matching monitors with a regex

### DISCUSSION THREAD

**Discussion by DJsushi** [hyprwm/Hyprland#7905] [General]
I work in an office where we have a lot of desks and a lot of monitors on those desks. When I come to the office, I just pick some random desk (whichever is free). But, I have to redo my Hyprland monitor setup because the monitor has a different name.
This is what my monitor config looks like right now:

```
monitor=desc:Philips Consumer Electronics Company PHL 275B1 UHB2347020690,2560x1440@74.97Hz,-320x-1440,1
monitor=desc:Philips Consumer Electronics Company PHL 275B1 UHB2347020698,2560x1440@74.97Hz,2240x-1440,1
monitor=desc:Philips Consumer Electronics Company PHL 275B1 UHB2347020699,2560x1440@74.97Hz,-320x-1440,1
monitor=desc:Philips Consumer Electronics Company PHL 275B1 UHB2347020695,2560x1440@74.97Hz,-320x-1440,1
monitor=desc:Philips Consumer Electronics Company PHL 275B1 UHB2347020299,2560x1440@74.97Hz,-320x-1440,1
```

As you can see, all the monitors are of the same brand, same model, just different serial numbers. It would be super nice to be able to match them with a single regex and my laptop automatically adjusting to the correct monitor setup.
I also have this exact same problem in my school, where we also have a bunch of the same monitor model sitting on different tables.

**Comment by vaxerski** [comment]
desc: is basically `what you input.*`
`desc:Philips Consumer Electronics Company PHL 275B1` should match all of them, unless someone else removed that since I wrote it

**Reply by DJsushi** [reply]
Thank you for the response. Interesting, I didn't know that. However, for some more complex patterns, wouldn't it be nicer to have a Regex there?

**Reply by vaxerski** [reply]
no
